docs/advisories/cve_2017_5715.md

```diff
@@ -0,0 +1,78 @@
+## CVE-2017-5715
+
+| NAME         	| CVE-2017-5715                                                                                                                                                                                          	|
+|--------------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Description  	| Systems with microprocessors utilizing speculative execution and branch prediction may allow unauthorized disclosure of information to an attacker with local user access via a side-channel analysis. 	|
+| Source       	| https://nvd.nist.gov/vuln/detail/CVE-2017-5715                                                                                                              |
+| NVD Severity 	| medium (attack range: local) |
+| Last Updated  | Jan 01 2018 |
+
+##  Solutions
+
+This hardware exploit requires the installation of the Linux Kernel >= 4.4.110.
+kops is currently running the 4.4.x kernel.  A rolling-update or replacement of
+the kops ami image, with kops hosted on AWS is recommended. All platforms are
+affected not just AWS.
+
+### Impacted kops / kubernetes Components
+
+- kops ami
+
+Other Linux distributions are impacted as well, please update with the
+appropriate AMI / Image.
+
+### Fixed Versions
+
+The following AMIs contain an updated kernel.
+
+- kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2018-01-05
+- kope.io/k8s-1.6-debian-jessie-amd64-hvm-ebs-2018-01-05
+- kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2018-01-05
+- kope.io/k8s-1.8-debian-jessie-amd64-hvm-ebs-2018-01-05
+- kope.io/k8s-1.8-debian-stretch-amd64-hvm-ebs-2018-01-05
+- kope.io/k8s-1.8-debian-stretch-amd64-hvm-ebs-2018-01-05
+
+These are the images that are maintained by the kops project, please refer to
+other vendors for the appropriate AMI version.
+
+### Update Process
+
+For all examples please replace `$CLUSTER` with the appropriate kops cluster
+name.
+
+Determine which instance groups exist for a cluster.
+
+`kops get ig --name $CLUSTER`
+
+Edit the kops instance groups with the appropriate image version via a `kops
+edit` command or `kops replace -f mycluster.yaml`.
+
+#### Perform dry-run update, verifying that all instance groups are updated.
+
+`kops update $CLUSTER` 
+
+#### Update the cluster.
+
+`kops update $CLUSTER --yes`
+
+Perform a dry-run rolling-update, verifying that all intance groups will be
```

Comment:
We switched from `####` to inline here.

Maybe do this for each:

```
### Dry-run update
Perform dry-run update, verifying that all instance groups are updated:
`kops update $CLUSTER` 
```

i.e. repeat, but keep headline short.